tox.ini

```diff
@@ -0,0 +1,60 @@
+; a generative tox configuration, see: https://tox.readthedocs.io/en/latest/config.html#generative-envlist
+
```

Comment:
all of these envs can be run independently.

For example, to just do a check do `tox -e check`